fluxacademy.html

```diff
@@ -0,0 +1,204 @@
+            <ul>
+                <li>Processes & techniques to create beautiful, modern websites</li>
+                <li>Strategies to provide more value to web design client</li>
+                <li>Roadmaps to a sustainable business as a freelance web designer</li>
+                <li>Tactics to manage projects and serve clients like a pro.</li>
+            </ul>
+                <!-- img needs to be in vertical pieces then become one as you scroll into view -->
+                <img src="" alt="course instructor image">
+        </div>   
+    </div>
+    <div>
+        <h1>Join 3,000+ Professional Designers in the Flux Community</h1>
+        <!-- img has animation when user touch individual pieces -->
+        <img src="" alt="picture layed out in a web of community members">
+        <p>Our global community is proof that you can be a successful freelancer no matter where you live. Colloborate, network, and continue learning from the experiences of your peers</p>
+    </div>
+    <div>
+        <!-- div has wave animation on top and bottom border -->
+        <img src= "" alt= "pattern triangle">
+        <h3>FREE WEB DESIGN COURSE IN YOUR INBOX</h3>
+        <p>Start learning with Ryan's free email course, High-Value WEb Designer Secrets. Sign up and discover the 3 most important lessons he learned as a freelance web designer</p>
+        <br>
+        <form>
+            <input type="text" name="first name" placeholder="First name">
+            <input type="text" name= "your email" placeholder="Email">
+            <button>Start my free course now!</button>
+        </form>
+        <a href="url">How we protect and manage your data</a>
+        <img src="" alt="pattern triangle">
+    </div>
+    <div class="header">
+        <h3>EXPERT RESOURCES FOR WEBSITE DESIGNERS</h3>
+    </div>
+    <div class="container">
+        <div class="left-col">
+            <img src="" alt="picture of greater and less than symbol with a line tilting to right between them">
+            <h5>HOW TO DO WEB DESIGN WITHOUT CODING</h5>
+            <p>It's the ultimate question. Can you become a web designer without learning how to  code? The short answer is yes. In this article, we'll dive into the best ways to approach how you can do web design without coding.</p>
+    </div>
+    <div class="container">
+        <div class="center-col">
+            <img src="" alt="a green checkmark in one box and an x in a red box">
+            <h5>WEBFLOW VERSUS WORDPRESS: WHICH ONE IS BETTER?</h5>
+            <p>39% of all websites on the Internet uses WordPress. While it may be a familiar platform, there are better solutions out there for creating websites. In this article, we'll dive deep into the difference between WordPress and Webflow</p>
+        
```

Comment:
Why is there an extra line here?

Reply:
H, because I was still figuring out how to do this section, I believe in JIRA I ask you to ignore this section.